### DISCUSSION TITLE
`x-vercel-deployment-url` missing in `vercel dev` after upgrading to Vercel CLI 19.1.1

### DISCUSSION THREAD

**Discussion by jasondonnette** [vercel/vercel#4729] [Help] [answered]
I just upgraded from Now CLI (`19.0.1`, I believe it was) to Vercel CLI `19.1.1` and when I run `vc dev` the `req.headers['x-vercel-deployment-url']` variable is missing.

I use this deployment URL when making serverless function-to-function calls.

Is this a known regression? (Also, should I be using some other technique to get this URL or is this still correct?)

**Comment by TooTallNate** [comment, marked as answer]
Hey @jasondonnette, thanks for the report. This is indeed a regression, will get that fixed ASAP.

**Reply by TooTallNate** [reply]
Fixed by https://github.com/vercel/vercel/pull/4850.

**Comment by y-nk** [comment]
@TooTallNate This still happening when testing a PUT endpoint with postman (maybe some postman settings?) (CLI v20.1.2 dev)
